apps/daimo-web/public/.well-known/apple-app-site-association

```diff
@@ -17,12 +18,14 @@
   },
   "activitycontinuation": {
     "apps": [
-      "JV8PYC9QV4.com.daimo"
+      "JV8PYC9QV4.com.daimo",
+      "JV8PYC9QV4.com.daimo.dev"
     ]
   },
   "webcredentials": {
     "apps": [
-      "JV8PYC9QV4.com.daimo"
+      "JV8PYC9QV4.com.daimo",
+      "JV8PYC9QV4.com.daimo.dev"
```

Comment:
If I'm understanding right

This means that daimo.xyz links will open in *either* the dev app or the production app

If you have both installed, does it prompt you?

Reply:
It picks the one higher up in the list if multiple are installed: https://developer.apple.com/forums/thread/649423